### PR TITLE
init documentation for `HTMLLinkElement.crossOrigin`

### LINKED ISSUE #21695: api.HTMLLinkElement - Broken URL for crossOrigin attribute
### What type of issue is this?

Other

### What information was incorrect, unhelpful, or incomplete?

Hi, 
I noticed a broken URL for the attribute crossOrigin in the compatibility table on page [](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement ). 

The broken link is currently:
`https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement/crossOrigin`

Possibly should be unless more relevant page exists:
`https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin`

All the best.

### What browsers does this problem apply to, if applicable?

_No response_

### What did you expect to see?

Correct working URL instead of 404.

### Did you test this? If so, how?

Clicked on the current link for crossOrigin

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.HTMLLinkElement`
* Report started: 2023-12-20T15:05:07.341Z

</details>

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

see the spec, https://html.spec.whatwg.org/multipage/semantics.html#attr-link-crossorigin and https://html.spec.whatwg.org/multipage/urls-and-fetching.html#cors-settings-attributes

also cc @TheOldMan2000

### Related issues and pull requests

Fixes https://github.com/mdn/browser-compat-data/issues/21695

### CODE REVIEW

File: files/en-us/web/api/htmllinkelement/crossorigin/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+{{APIRef("HTML DOM")}}
+
+The **`crossOrigin`** property of the {{domxref("HTMLLinkElement")}} interface specifies the Cross-Origin Resource Sharing ({{Glossary("CORS")}}) setting to use when retrieving the resource.
+
+## Value
+
+A string of a keyword specifying the CORS mode to use when fetching the resource. Available values are:
+
+- `anonymous` or an empty string (`""`)
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/crossorigin/index.md:17]:
```suggestion
- `anonymous` or the empty string (`""`)
```

File: files/en-us/web/api/htmllinkelement/crossorigin/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+browser-compat: api.HTMLLinkElement.crossOrigin
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`crossOrigin`** property of the {{domxref("HTMLLinkElement")}} interface specifies the Cross-Origin Resource Sharing ({{Glossary("CORS")}}) setting to use when retrieving the resource.
+
+## Value
+
+A string of a keyword specifying the CORS mode to use when fetching the resource. Available values are:
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/crossorigin/index.md:15]:
```suggestion
A string of a keyword specifying the CORS mode to use when fetching the resource. Possible values are:
```

File: files/en-us/web/api/htmllinkelement/crossorigin/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- `anonymous` or an empty string (`""`)
+  - : Requests sent by the {{domxref("HTMLLinkElement")}} will use the `cors` mode and the `same-origin` credentials mode. This means that CORS is enabled and credentials are sent _if_ the resource is fetched from the same origin from which the document was loaded.
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/crossorigin/index.md:18]: Can we link to something explaining mode, and credentials mode (like a property?)

File: files/en-us/web/api/htmllinkelement/crossorigin/index.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- `use-credentials`
+  - : Requests sent by the {{domxref("HTMLLinkElement")}} will use the `cors` mode and the `include` credentials mode. All resources requests by the element will use CORS, regardless of what domain the fetch is from.
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/crossorigin/index.md:20]: Same here

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by TheOldMan2000** [comment]
The credentials links on that page are all returning 404 errors. Same issue, document not written yet?

**Comment by skyclouds2001** [comment]
> The credentials links on that page are all returning 404 errors. Same issue, document not written yet?

Emmm, would you please list which link is inaccessible?

**Comment by TheOldMan2000** [comment]
Sorry,

Referring to all the credentials hyperlinks pointing to 
`https://pr31277.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/Request/credentials `

**Comment by skyclouds2001** [comment]
This is because the preview result only build the pages that has been changed, that is, only the pages that has been updated can view in the preview result

so in this case, since `HTMLLinkElement.crossOrigin` is the only page that has been updated, only `HTMLLinkElement.crossOrigin` can view, any other page cann't view

**Pull request merged**